src/algorithms/crypto/randomness.go

```diff
@@ -1,28 +0,0 @@
-package crypto
-
-import (
-	"bytes"
-	"encoding/binary"
-
-	addr "github.com/filecoin-project/go-address"
-	"github.com/minio/blake2b-simd"
-
-	abi "github.com/filecoin-project/specs-actors/actors/abi"
-	big "github.com/filecoin-project/specs-actors/actors/abi/big"
```

Comment:
Looks like `github.com/filecoin-project/specs-actors/actors` is replaced with vendored version at `src/actors` that doesn't have package `big`.

`github.com/filecoin-project/specs-actors` does have a package `abi/big`, so maybe this is what the author was referring to, but in this case we should probably remove `src/actors`.

Reply:
Having read what @anorth said about spec-actors being based on this repo, if `abi/big` is something needed for Filecoin (which I guess it is), we should probably copy it back to this repo as a `source-of-truth` - maybe in a follow up PR alongside the rest of the issues once identified.

Reply:
So I do not think we should copy pedestrian code like a bigint wrapper into the spec (but IMO the spec shouldn't really have much code anyway).

I'm not sure if it's obvious, but the submodule actors repo is just the same specs-actors repo. It's not "vendored" or behind on purpose, but because updating the code here to keep up with it is painful busywork.

It would be better if this repo did not have code that depended on the actors. It's probably ok if pragmatically it does so just to import a bunch of types like the `abi` package and bigint wrappers.